app/controllers/admin/emails/webmaster_questions_controller.rb

```diff
@@ -40,8 +40,8 @@ def create_webmaster_question
         elsif non_user_potential_spam?
           flash_error(:runtime_ask_webmaster_antispam.t)
         else
-          QueuedEmail::Webmaster.create_email(sender_email: @email,
-                                              content: @content)
+          QueuedEmail::Webmaster.create_email(@user, sender_email: @email,
+                                                     content: @content)
```

Comment:
If the `QueuedEmail` builder now gets the `@user`, I don't think it needs to get `sender_email` too, it can derive sender_email from `@user`. (see action `new` above, where `@email` is defined.)

Reply:
I was trying to do the simplest convention that I could.  The challenge with putting the user argument at the end are functions that slurp up the rest of the arguments.  This case is an example where just looking at the single line of code is is not clear that `@email` is always exactly `@user.email`.  I'll review and see if there are any real inconsistencies.

Reply:
Based on the way it's established in `new` it seems `@email` is always going to be `@user.email`. A caller could send the a `user` param, but even then...

Reply:
Yeah, can't do it in the simplest possible way.  There are several cases where the user.email is not the sender_email.  For example folks who aren't logged in but want to send a comment.  I made the `sender_email` argument default toe `user.email` when `user` is non-nil.  It cleaned up a few cases, but doesn't appear to be a huge win.